### DISCUSSION TITLE
안녕하세요 Ngrinder 테스트 시도하는데 로그인이 되지 않습니다.

### DISCUSSION THREAD

**Discussion by LeeJongbokz** [naver/ngrinder#920] [Q&A] [answered]
- 안녕하세요! Ngrinder 테스트 시도 하는데 로그인이 되지 않아서,
   해결 방법 여쭙고자 글 남깁니다.
   ID: admin, PW: admin 으로 로그인 시도했고,
   http://localhost:8080/login에서 시도했습니다!

**Comment by imbyungjun** [comment]
Is there any error logs on browser console?

Try to run `ngrinder-controller` with adding `--spring.h2.console.enabled=true` option. After adding the option, you can access to h2 console on URL `{ip}:{port}/h2-console`. Check the user ID and password in database.

**Reply by LeeJongbokz** [reply]
java -jar ngrinder-controller-3.5.6.war --spring.h2.console.enabled=true
-> 이렇게 작성하는 것이 아닐까요?
위와 같이 쓰니까 아래 이미지와 같은 에러가 나왔습니다ㅎㅎ
![image](https://user-images.githubusercontent.com/8718430/196399510-2c86a446-096a-4e80-90f2-28413cc1099a.png)

**Comment by junoyoon** [comment]
@imbyungjun 님 그냥 한국어로 물어보면 한국어로 대답해도 될것 같아요. github에 번역 기능도 들어왔구요.

**Comment by donggyu04** [comment, marked as answer]
@LeeJongbokz 님 

`{유저 홈}/.ngrinder` 폴더 들어가신뒤 아래 파일을 여시고, `controller.admin_password_reset=true` 추가해 주신후에 컨트롤러 재 시작하면
admin/admin 으로 어드민 사용자 계정 비밀번호 초기화 됩니다.

![image](https://user-images.githubusercontent.com/14273601/196618308-4d82cd03-cfd4-4b87-a0fe-30acd78994f2.png)

**Reply by LeeJongbokz** [reply]
답변 감사드립니다!
